Library/OcDevicePathLib/OcDevicePathLib.c

```diff
@@ -1277,7 +1277,7 @@ OcDevicePathHasFilePathSuffix (
     return FALSE;
   }
 
-  PathNameLen = OcFileDevicePathFullNameLen (&FilePath->Header);
+  PathNameLen = OcFileDevicePathFullNameLen (&FilePath->Header) - 1;
```

Comment:
This can now wrap around and bork the check below. Please purge the subtraction, change the operator below (<=) and remove the addition further down.

Reply:
Thx - I probably shouldn't really have changed `OcFileDevicePathFullNameLen` from producing a normal strlen value (not including terminator); I'll fix it by changing that instead.

Reply:
Believe all correct now - https://github.com/acidanthera/OpenCorePkg/commit/991cd3b2dce69823a25803f0773306e691917a21

Sorry about that - lazy! - tyvm for review